### PR TITLE
Lnd create

### PR DESCRIPTION
**Pull request by marcosrdz**
ADD: You can now pay for a Lightning invoice by manually inserting the invoice ID.

### CODE REVIEW

**Comment by Overtorment** [comment]
did u branch out from `development`..?
too many commits in branch

**Comment by marcosrdz** [comment]
Yes. I did.

**Comment by Overtorment** [comment]
Next time branch out of master

File: MainBottomTabs.js [hunk 1]
```diff
@@ -135,6 +135,9 @@ const Tabs = createStackNavigator(
     ScanLndInvoice: {
       screen: ScanLndInvoice,
     },
+    ScanQrAddress: {
+      screen: sendScanQrAddress,
```
Comment (Overtorment) [inline MainBottomTabs.js]: cant figure out why is it here?

Comment (marcosrdz) [reply MainBottomTabs.js]: ScanLNDInvoice had it's own BarCodeScanner in render(). Had to put that in MainBottomsTabs so ScanLndInvoice could reuse the existing one.

**Pull request merged**